dev/com.ibm.ws.jpa.container.v31/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,12 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.methodParameters=do not generate
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.8
```

Comment:
Since Jakarta EE 10 requires Java 11 as a minimum, this project should probably specify 11 for target, compliance, and source

Reply:
Meaning these should say `=11` instead of `=1.8`